CHANGELOG.md

```diff
@@ -74,8 +74,10 @@
 
 ## Other changes
 
-- Adding an annotation to function (`'info` type). Useful to store result of analysis. (see [[#1016](https://github.com/jasmin-lang/jasmin/issues/1016)])
-- ([PR#1021](https://github.com/jasmin-lang/jasmin/issues/1021))
+- Add an annotation to functions (of type `'info`); this can be useful to store
+  the result of an analysis
+  ([PR#1021](https://github.com/jasmin-lang/jasmin/pull/1021);
+  fixes [#1016](https://github.com/jasmin-lang/jasmin/issues/1016)).
```

Comment:
Already fixed in `release-2024.07`; to be merged in `main` after the next minor release.

Reply:
Indeed, but since I like being picky, there are two issues with what is in `release-2024.07`:
- the URL to the PR contains `issues` (not that bad, since GitHub redirects correctly)
- the shortname of the link to the issue uses "PR"